src/EnergyPlus/CondenserLoopTowers.cc

```diff
@@ -105,59 +107,20 @@ namespace CondenserLoopTowers {
     // PURPOSE OF THIS MODULE:
     // Model the performance of cooling towers
 
-    // Empirical Model Type
-    int const CoolToolsXFModel(1);
-    int const CoolToolsUserDefined(3);
-    int const YorkCalcModel(4);
-    int const YorkCalcUserDefined(5);
-
-    int const EvapLossByUserFactor(80);
-    int const EvapLossByMoistTheory(81);
-
-    int const BlowdownByConcentration(90);
-    int const BlowdownBySchedule(91);
-
     std::string const cCoolingTower_SingleSpeed("CoolingTower:SingleSpeed");
     std::string const cCoolingTower_TwoSpeed("CoolingTower:TwoSpeed");
     std::string const cCoolingTower_VariableSpeed("CoolingTower:VariableSpeed");
     std::string const cCoolingTower_VariableSpeedMerkel("CoolingTower:VariableSpeed:Merkel");
```

Comment:
It's complaining when I try to move this to ```const char *```. For example:

> ShowSevereError(cCoolingTower_SingleSpeed + " \\"" + this->Name + "\\". Free convection air flow rate must be less than the design air flow rate.")

```Invalid operands ('const char *' and 'const char [3]')```

Any suggestions?

Reply:
So the reason it doesn't work is because the arguments are fundamentally pointers, and there is no addition for pointers.  You have a couple options.  You can make them const char and then when they are used in error message expressions later, you can create a new string from the char array:  `std::string(cCoolingTowerString) + std::string(something)`.  There is surely a penalty associated with that, but if it is only for error reporting then that is acceptable.